src/app/datepicker/datepicker.demo.ts

```diff
@@ -88,4 +92,16 @@ export class DatepickerDemoComponent implements OnInit {
     this.datepicker.readonly = true;
     this.datepickerReadOnly = this.datepicker.readonly;
   }
+
+  registerCustomValidator() {
+    $.fn['validation']['rules']['customRule'] = {
+      check: function (value: any, field: any, grid: any) {
+        console.log(value, field, grid);
+        return false;
+      },
+      id: 'custom',
+      type: 'error',
+      message: 'Test Error - Anything you enter will be wrong'
+    };
+  }
```

Comment:
just curious, but is there a way perform the register without using jQuery?

Reply:
Yes there is using Soho.validation.rules I think that’s better I’ll try and push an update